downstream/modules/automation-hub/proc-create-synclist.adoc

```diff
@@ -12,11 +12,22 @@ By default, all Red Hat Certified collections are included in your initial organ
 
 * You have a valid Ansible Automation Platform subscription.
 * You have Organization Administrator permissions for cloud.redhat.com.
+* Ensure that the following domain names are part of either the firewall or the proxy's allowlist for successful connection and download of collections from {HubName} or Galaxy server:
+** `galaxy.ansible.com`
+** `cloud.redhat.com`
+** `console.redhat.com`
+** `sso.redhat.com`
+* {HubNameStart} resources are stored in Amazon Simple Storage.
```

Comment:
{HubName} and {HubNameStart} aren't rendering properly in my local docs build. I think we need to create a PR to main to fix this.

Try adding the following line to downstream/titles/hub/configuring-private-hub-rh-certified/master.adoc

```
include::attributes/attributes.adoc[]
```